### PR TITLE
Implement multiple acronyms in !!acro and UQ parking with !parking

### PR DESCRIPTION
**Pull request by JosephGarrone**
Can fetch multiple acronyms

### CODE REVIEW

**Comment by JosephGarrone** [comment]
Can now fetch the parking with !parking.

**Comment by JosephGarrone** [comment]
Added some aliases to the car park names

File: scripts/acronym.js [hunk 1]
```diff
@@ -7,22 +7,69 @@
...
+  // Parses the DOM to get the acroynm expansion
+  robot.getAcronym = function (html) {
+    var $ = cheerio.load(html);
+    var responses = [];
+    
+    $('td.acr').each(function (i, element) {
+      responses.push($(element).siblings().text());
+    });
+    
+    return responses[Math.floor(Math.random() * responses.length)];
```
Comment (TRManderson) [inline scripts/acronym.js]: I still have mixed feelings about choosing randomly, rather than taking the first

Comment (JosephGarrone) [inline scripts/acronym.js]: This is like mon's im pretty sure, ensure that when people acro multiple times it is never the same.

Comment (TRManderson) [inline scripts/acronym.js]: Choosing from the top 10 maybe?

Comment (JosephGarrone) [inline scripts/acronym.js]: There are generally only about 5 results anyway

**Pull request merged**